package.json

```
@@ -1194,12 +1194,15 @@
                     "launch": {
                         "properties": {
                             "args": {
-                                "default": [],
-                                "description": "Command line arguments passed to the program",
+                                "default": "",
+                                "description": "Command line arguments passed to the program. (not supported for `internalConsole`)",
```

```suggestion
                                "default": [],
                                "description": "Command line arguments passed to the program.",
```
@int19h  We don't need to say that it is not supported in internal console, right? and the default should still be `[]`.

---

"args" in general is supported, only the no-array version isn't. I think this is enough of a corner case to not bother mentioning in the docstring.